app/main.py

```diff
@@ -1,3 +1,51 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, value: Distance | int) -> Distance:
```

Comment:
`value` can be float too. Fix everywhere, please
```suggestion
    def __add__(self, value: Distance | int | float) -> Distance:
```